src/com/amazon/ionelement/api/IonUtils.kt

```diff
@@ -52,12 +52,12 @@ fun IonValue.toIonElement(): AnyElement =
         createIonElementLoader().loadSingleElement(reader)
     }
 
-/** Throws an [IonElectrolyteException], including the [IonLocation] (if available). */
-fun ionError(blame: IonElement, description: String): Nothing {
-    ionError(blame.metas, description)
+/** Throws an [IonElementException], including the [IonLocation] (if available). */
+internal fun constraintError(blame: IonElement, description: String): Nothing {
+    constraintError(blame.metas, description)
 }
 
-/** Throws an [IonElectrolyteException], including the [IonLocation] (if available). */
-fun ionError(metas: MetaContainer, description: String): Nothing {
-    throw IonElectrolyteException(metas.location, description)
+/** Throws an [IonElementException], including the [IonLocation] (if available). */
+internal fun constraintError(metas: MetaContainer, description: String): Nothing {
```

Comment:
This used to be an exposed function or is my Kotlin knowledge failing me again?

Reply:
it was part of the API previously, but I doubt anyone is using it.  Even so, I am breaking so many other things with `v0.2.0` I thought:  "what's one more"?  

In any case, I don't know if it's all that useful to provide an API to throw an `IonElementConstraintException` from client code and I erred on the side of a smaller API.  I could remove the `internal`, if you think it's helpful.

Reply:
No it's fine as is